tasks/main.yml

```diff
@@ -199,6 +201,8 @@
 - name: Configure PAM
   ansible.builtin.import_tasks:
     file: password.yml
+  when:
```

Comment:
That's to broad, `password.yml` configures various password related things, not just PAM, and `manage_pam` is already set on those tasks.